server/Cargo.toml

```diff
@@ -4,10 +4,17 @@ version = "0.1.0"
 description = "gitkv is a server for using git as a key value store for text files"
 authors = ["Intent HQ <engineering@intenthq.com>"]
 edition = "2018"
+license = "MIT"
```

Comment:
We should add the `license` on all of the other `Cargo.toml` too, even if we're not publishing them to `crates.io`.